Orange/base.py

```diff
@@ -101,7 +102,7 @@ def fit_storage(self, data):
         X, Y, W = data.X, data.Y, data.W if data.has_weights() else None
         return self.fit(X, Y, W)
 
-    def __call__(self, data):
+    def __call__(self, data, callback=None):
```

Comment:
I glanced over this PR. The only thing I don't like is the name of the argument. `callback` can be anything. Perhaps `progress_callback`? Longer, but more informative.